package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "register_a_food_business_front_end",
-  "version": "10.0.1",
+  "version": "9.5.0",
```

Comment:
Revert these changes

Reply:
Reverted these changes and upgraded packages that are causing snyk failures. Snyk only seems to report on master branch however so won't reflect the updates in the checks yet.